rss/erc-last-call.xml

```diff
@@ -0,0 +1,35 @@
+---
+layout: null
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
+  <channel>
+    <title>Ethereum EIPs - Last Call Review</title>
+    <description>All EIPs which are in the two-week "last call" status, please help review these and provide your feedback!</description>
```

Comment:
```suggestion
    <description>All EIPs which are in the "last call" status, please help review these and provide your feedback!</description>
```

Technically not always two weeks.

Reply:
Addressed at a2534bfb77d80b6e4755f5ec0b8ba04b5d921cfb